ServerRules.md

```diff
@@ -0,0 +1,11 @@
+## Server Rules
+
+1. Don't do anything you wouldn't do in Oso Negro
+2. Read and follow the Code of Conduct (https://nelson.social/about/more#code-of-conduct)
+3. No racist, sexist, homophobic, transphobic, ableist, or otherwise discriminatory or hateful jokes or language
+4. Don't post or threaten to post other people’s personally identifying information (i.e., doxing)
```

Comment:
Do we need to say this?

Reply:
It's in the https://nelson.social/about/more#unacceptable-behavior list so I guess not, unless we think it's particularly important to call out?

Reply:
I don't think it is.

Reply:
Please are going to scan the rules and I think we should set the tone of nelson.social, not list all the various things that are anyhow illegal in BC or already unspoken rules.

Reply:
I'd be happy to lose rules 3-7 as they're just unpacking rule 2.